src/session.rs

```diff
@@ -198,7 +206,12 @@ pub struct SessionUnmounter {
 
 impl SessionUnmounter {
     /// Unmount the filesystem
+    /// # Errors
+    /// Does not error.
```

Comment:
Let's remove this. I don't think we should document the function as not returning an error, especially given the TODO to make it return an error